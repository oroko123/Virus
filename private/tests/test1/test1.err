Error in line 2:znaczek   polski
Error in line 15:Trawa 2 1971 Poczta Polska
Error in line 17:2000 1999
