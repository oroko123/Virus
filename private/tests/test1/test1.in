Franklin Z-Grill ONE CENT 930000 1867 U.S.POSTAGE
znaczek   polski
One Cent Black on Magenta ONE CENT  935000 1856  British Guiana
One Penny 1072260 1847 Mauritius
 Two Pence 1148850 1847 Mauritius
Inverted Jenny 24 CENTS 977500 1918 U.S.POSTAGE
Three-Skilling Yellow 2300000 1855 SVERIGE
Pomidor 1 1971 Poczta Polska
Brzoza 1,50 1971 Poczta Polska
Buk 0,50 1971 Poczta Polska
Dynia 1 1971 Poczta Polska
queen's slipper 3.00 1999 Kanada
1855 1856
1847 1847
Trawa 2 1971 Poczta Polska
1970 1972
2000 1999
