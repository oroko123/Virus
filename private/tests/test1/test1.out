1855 SVERIGE 2300000 Three-Skilling Yellow
1856 British Guiana 935000 One Cent Black on Magenta ONE CENT
1847 Mauritius 1072260 One Penny
1847 Mauritius 1148850 Two Pence
1971 Poczta Polska 0,50 Buk
1971 Poczta Polska 1 Dynia
1971 Poczta Polska 1 Pomidor
1971 Poczta Polska 1,50 Brzoza
